runtimes/browser.js

```diff
@@ -20,6 +20,12 @@ var $ = window.$ = {
     var fwin = frame.contentWindow;
     var fdoc = fwin.document;
     var fscript = fdoc.createElement('script');
+
+    // The following is a workaround for a bug in Chromium related to reporting
+    // errors produced from evaluating code using `eval`.
+    // https://bugs.chromium.org/p/chromium/issues/detail?id=746564
+    fdoc.write('<body>');
```

Comment:
I'm glad you reminded me of that... unfortunately, it also means that I won't land this until @bterlson has a chance to weigh in.